src/main/resources/org/sagebionetworks/web/client/widget/entity/annotation/EditAnnotationsDialogViewImpl.ui.xml

```diff
@@ -3,45 +3,36 @@
 	xmlns:b="urn:import:org.gwtbootstrap3.client.ui"
 	xmlns:b.html="urn:import:org.gwtbootstrap3.client.ui.html"
 	xmlns:t="urn:import:org.sagebionetworks.web.client.view.bootstrap.table"
+	xmlns:s="urn:import:org.sagebionetworks.web.client.widget.entity.controller"
 	xmlns:w="urn:import:org.sagebionetworks.web.client.widget">
 
 	<g:HTMLPanel>
-		<b:Modal ui:field="editModal"
-			closable="true" dataBackdrop="STATIC" dataKeyboard="true"
+		<b:Modal ui:field="editModal" title="Edit Annotations"
+			closable="true" dataBackdrop="STATIC" dataKeyboard="true" size="LARGE"
 			b:id="editAnnotationsModal">
 			<b:ModalHeader>
-				<b:Heading size="H4" addStyleNames="displayInline" text="Edit Annotations" />
```

Comment:
Like the modal header style change, but worried that other modals in the app will not get it (because they use this other pattern of explicitly defining a Heading in the ModalHeader).
Please revert this specific change, and add a new css selector to fix it too (in addition to the css that fixes the Modal setTitle case).